ios/Classes/Model/Peripheral.m

```diff
@@ -1,5 +1,5 @@
 #import "Peripheral.h"
-#import "PeripheralResponse.h"
+#import "PeripheralResponse1.h"
```

Comment:
BlemulatorPeripheralResponse?

Reply:
Changed according to the review suggestion.